packages/app/src/app/pages/SignIn/SignIn.tsx

```diff
@@ -22,8 +22,20 @@ export const SignIn = ({ redirectTo, onSignIn }: SignInProps) => {
     }
   }, [getPendingUser, pendingUserId]);
 
-  if (duplicateAccountStatus) {
-    return <DuplicateAccount provider={duplicateAccountStatus.provider} />;
+  /**
+   * 🚧 Utility to debug Duplicate Account
+   */
+  const DA_DEBUG = window.localStorage.getItem('DA_DEBUG') === 'ENABLED';
```

Comment:
We now have the ability to debug the duplicate account view without having to jump through hoops creating new accounts. Right now this is out of scope for the next release, but we want to update the screen after.